test/unit/DssVestTopUp.test.ts

```diff
@@ -105,6 +117,192 @@ describe("DssVestTopUp", function () {
     );
   });
 
+  describe("Deploy", function () {
```

Comment:
What do you think about adding one test case to check if all params are set correctly when it deploys without revert?